src/index.html

```diff
@@ -1,16 +1,56 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="wrapper">
+      <div class="radius">
+        <form class="form_big"
+          autofocus
+          action="value"
+          method="get"
+          data-qa="big"
+        >
+          <button class="button_big" type="submit">
+            <img class="img_big"
+              src="./images/Search.svg" alt="img"
+            >
+          </button>
+          <input class="input_big"
+            name="s"
+            placeholder="Try &#34Los Angeles&#34"
+            type="search"
+            data-qa="keypress"
+          >
+        </form>
+      </div>
+      <div class="radius_small">
+        <form class="form_small"
+          action="value"
+          method="get"
+          data-qa="small"
+        >
+          <button class="button_small" type="submit">
+            <img class="img_small"
+              src="./images/Search.svg" alt="img"
+            >
+          </button>
+          <input class="input_small"
+            name="s"
+            placeholder="Try &#34Los Angeles&#34"
+            type="search"
+            data-qa="small"
+            >
```

Comment:
Pay attention on indents
```suggestion
          <input class="input_small"
            name="s"
            placeholder="Try &#34Los Angeles&#34"
            type="search"
            data-qa="small"
          >
```